datadog/runchecks.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+echo `date`;
+. ~/.bashrc;
+~/q/l32/q ./datadog/datachecks.q
```

Comment:
these hardcoded paths seem like they could be problematic

Reply:
Removed hard coding by adding PATH to crontab in setup file and removed .bashrc as this is not needed.